functions/local-source/README.md

```diff
@@ -0,0 +1,60 @@
+# Build with local source code
```

Comment:
I suggest putting the content in readme to https://openfunction.dev/docs/concepts/function_build/ as another build method. And then reference this sample as a link